src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -33,7 +33,10 @@
      * @throws IllegalValueException if any of the raw values are invalid
      */
     public AddCommand(String name, String priorityValue) throws IllegalValueException {
-        this.toAdd = new FloatingTask(new Name(name), new Priority(priorityValue));
+        if (priorityValue != null)
+            this.toAdd = new FloatingTask(new Name(name), new Priority(priorityValue));
+        else
+            this.toAdd = new FloatingTask(new Name(name));
```

Comment:
Maybe null not that good but since the default value string constant is in floating task class bobian